roles/workstation/files/xscreensaver-config

```diff
@@ -0,0 +1,256 @@
+# XScreenSaver Preferences File
```

Comment:
I would be a fan of aligning this stuff with spaces.  Emacs has a [thing](https://www.emacswiki.org/emacs/AlignCommands) for it, but it's a bit messier than [vim's](https://github.com/junegunn/vim-easy-align) 😮.  Just while we're messing up blame.